tests/robottelo/test_helpers.py

```diff
@@ -66,18 +71,23 @@ def test_cat_fail(self, ssh):
             return_value=FakeSSHResult([], 1, 'stderr'))
         with self.assertRaises(HostInfoError) as context:
             get_host_info()
+        if six.PY2:
+            message = context.exception.message
+        else:
+            message = str(context.exception)
```

Comment:
Is there any advantage to accessing the `message` attribute directly? Coercing an exception to a string should be sufficient, no?

```python
>>> def foo():
...   raise Exception('some custom message')
... 
>>> try:
...   foo()
... except Exception as err:
...   error_message = str(err)
... 
>>> error_message
'some custom message'
>>> print(error_message)
some custom message
>>> type(error_message)
<type 'str'>
```